### PR TITLE
Refactor keymapping for resident_runner

### PR DESCRIPTION
**Pull request by dnfield**
I was going to look at adding another key to this mapping and was pretty confused by all the if/else if/lower/character logic.

This converts it to a switch that is alphabetized, which makes it easier to determine if a character is already in use or not when adding new ones.

No tests because no new behavior.

### CODE REVIEW

File: packages/flutter_tools/lib/src/resident_runner.dart [hunk 1]
```diff
@@ -807,78 +807,90 @@ abstract class ResidentRunner {
 
   /// Returns [true] if the input has been handled by this function.
   Future<bool> _commonTerminalInputHandler(String character) async {
-    final String lower = character.toLowerCase();
```
Comment (christopherfujino) [inline packages/flutter_tools/lib/src/resident_runner.dart]: looks like you lost the `toLowerCase()`

Comment (christopherfujino) [reply packages/flutter_tools/lib/src/resident_runner.dart]: oh, nvm

File: packages/flutter_tools/lib/src/resident_runner.dart [hunk 2]
```diff
@@ -807,78 +807,90 @@ abstract class ResidentRunner {
...
+        }
+        return false;
+      case 'd':
+        await detach();
         return true;
-      }
-    } else if (character == 'S') {
-      if (supportsServiceProtocol) {
-        await _debugDumpSemanticsTreeInTraversalOrder();
+      case 'h':
```
Comment (christopherfujino) [inline packages/flutter_tools/lib/src/resident_runner.dart]: looks like `'H'` used to work here

File: packages/flutter_tools/lib/src/resident_runner.dart [hunk 3]
```diff
@@ -807,78 +807,90 @@ abstract class ResidentRunner {
...
+          return true;
+        }
+        return false;
+      case 'U':
+        if (supportsServiceProtocol) {
+          await _debugDumpSemanticsTreeInInverseHitTestOrder();
+          return true;
+        }
+        return false;
+      case 'w':
```
Comment (christopherfujino) [inline packages/flutter_tools/lib/src/resident_runner.dart]: same for `'W'`

Comment (christopherfujino) [reply packages/flutter_tools/lib/src/resident_runner.dart]: et. al

**Review by christopherfujino — APPROVED** [review]
lgtm

**Review by jonahwilliams — APPROVED** [review]
LGTM

**Pull request merged**